src/nodes/core/ContextNode.js

```diff
@@ -138,5 +138,7 @@ export const context = /*@__PURE__*/ nodeProxy( ContextNode );
  */
 export const label = ( node, name ) => context( node, { label: name } );
 
-addMethodChaining( 'context', context );
-addMethodChaining( 'label', label );
+addMethodsChaining( {
```

Comment:
TBH, I find the previous more explicit version easier to read.

More compact code is not necessary something we are heading for. If you use a build system with a minifier, the improvements in size do usually not matter.

Reply:
Just to simplily test it out, the code minify looks like:

code

```ts
import { addMethodChaining } from "three/tsl";

// this method does not in r171, we add it here.
function addMethodsChaining(object: Record<string, unknown>) {
    for (const key in object) {
        addMethodChaining(key, object[key]);
    }
}

const NOOP = () => {};

addMethodChaining( 'test', NOOP );
addMethodChaining( 'dev', NOOP );

addMethodsChaining({
    test: NOOP,
    dev: NOOP
});
```

minify result:

```js
function TN(e){for(const t in e)Fo(t,e[t])}const Yi=()=>{};Fo("test",Yi),Fo("dev",Yi),TN({test:Yi,dev:Yi});
```

`Fo("test",Yi),Fo("dev",Yi)` is longer than `TN({test:Yi,dev:Yi})`

Reply:
I compared this logic with exporter, sometimes it can be useful to have `export { a, b c }` instead of `export a; export b; export c;`

Reply:
> TBH, I find the previous more explicit version easier to read.

+1

Reply:
@Mugen87 @mrdoob If you both prefer I will revert it in this release.

Reply:
Yes please 🙏